tests/test_ops.py

```diff
@@ -1529,6 +1565,26 @@ def test_mathtransform():
+        # {"opName": "max", "outName": "max/rank2_float64", "varShapes": [[2, 12], [2, 12]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "min", "outName": "min/rank2_float64", "varShapes": [[2, 12], [2, 12]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "mod", "outName": "mod/rank2_float64", "varShapes": [[2, 12], [2, 12]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank2_int32", "varShapes": [[2, 12], [2, 12]], "varTypes": ["int32", "int32"], "varInit": ["uniform_int10", "uniform_in10"]},
+        # {"opName": "min", "outName": "min/rank2_int32", "varShapes": [[2, 12], [2, 12]], "varTypes": ["int32", "int32"], "varInit": ["uniform_int10", "uniform_int10"]},
+        # {"opName": "mod", "outName": "mod/rank2_int64", "varShapes": [[2, 12], [2, 12]], "varTypes": ["int64", "int64"], "varInit": ["uniform_in10", "uniform_int10"]},
+        # {"opName": "mod", "outName": "mod/rank2_float32", "varShapes": [[2, 12], [2, 12]], "varTypes": ["float32", "float32"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "mod", "outName": "mod/rank2_float64", "varShapes": [[2, 12], [2, 12]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_int32", "varShapes":[[2,3,10], [2,3,10]], "varTypes":["int32", "int32"], "varInit":["uniform_int10", "uniform_int10"]},
+        # {"opName": "max", "outName": "max/rank3_float32", "varShapes": [[2, 3, 12], [2, 3, 12]], "varTypes": ["float32", "float32"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_float32", "varShapes": [[2, 3, 12], [2, 3, 12]], "varTypes": ["float32", "float32"],  "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_float64", "varShapes": [[2, 3, 10], [2, 3, 10]],  "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_float64", "varShapes": [[2, 3, 12], [2, 3, 12]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_float64", "varShapes": [[2, 3, 12], [2, 3, 12]], "varTypes": ["float64", "float64"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_half", "varShapes": [[2, 3, 10], [2, 3, 10]],  "varTypes": ["half", "half"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_half", "varShapes": [[2, 3, 12], [2, 3, 12]], "varTypes": ["half", "half"], "varInit": ["uniform", "uniform"]},
+        # {"opName": "max", "outName": "max/rank3_half", "varShapes": [[2, 3, 12], [2, 3, 12]], "varTypes": ["half", "half"], "varInit": ["uniform", "uniform"]},
```

Comment:
No broadcast cases, we'll want at least some of those. You can just change some of the shapes for the existing test cases here
For example `[[2, 1, 10], [2, 3, 1]]` (to give [2,3,10] output shape), or `[[], [2,12]` for scalar broadcast

> NOTE: math.maximum supports broadcasting.

https://www.tensorflow.org/api_docs/python/tf/math/maximum?hl=sl

Reply:
Added a few cases.